source/delegate/migrations/1_initial_migration.js

```diff
@@ -1,12 +1,16 @@
+const DelegateFactory = artifacts.require('DelegateFactory')
 const Delegate = artifacts.require('Delegate')
 
-module.exports = (deployer, network) => {
-  if (network == 'rinkeby' || network == 'mainnet') {
+module.exports = async(deployer, network) => {
     // fill in the addresses below
     const SWAP_ADDRESS = ''
     const INDEXER_ADDRESS = ''
     const OWNER_ADDRESS = ''
     const TRADE_WALLET_ADDRESS = ''
-    deployer.deploy(Delegate, SWAP_ADDRESS, INDEXER_ADDRESS, OWNER_ADDRESS, TRADE_WALLET_ADDRESS)
+    const FACTORY_PROTOCOL = ''
```

Comment:
Should we fill this in? I feel like we have a standard factory protocol that we use

Reply:
I believe this is just for dummy verification for the deployed delegate contact. So, it's ok for it to be empty.

Reply:
Happy to put in '0x0001' as that's what in the deployer repo as well.